MediaBrowser.Providers/Plugins/MusicBrainz/MusicBrainzRecordingId.cs

```diff
@@ -0,0 +1,27 @@
+using MediaBrowser.Controller.Entities.Audio;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.Providers;
+
+namespace MediaBrowser.Providers.Plugins.MusicBrainz;
+
+/// <summary>
+/// MusicBrainz recording id.
+/// </summary>
+public class MusicBrainzRecordingId : IExternalId
+{
+    /// <inheritdoc />
+    public string ProviderName => "MusicBrainz";
+
+    /// <inheritdoc />
+    public string Key => MetadataProvider.MusicBrainzRecording.ToString();
+
+    /// <inheritdoc />
+    public ExternalIdMediaType? Type => ExternalIdMediaType.Recording;
+
+    /// <inheritdoc />
+    public string UrlFormatString => Plugin.Instance!.Configuration.Server + "/recording/{0}";
```

Comment:
Will the metadata fetcher work correctly with this URL?

Reply:
The URL is similar to all other MusicBrainz URLs, just with `recording` instead of for example `track`.

For example, `All the Small Things` recording and one of the specific track:
https://musicbrainz.org/recording/f36d9818-019b-4379-ad13-5080feb9ad8a
https://musicbrainz.org/track/27681a20-3aab-30b8-ab45-7fd274e6dfd5

Also, I have not implemented an `IRemoteMetadataProvider` for this as I don't think there is any value in that. But if implemented, it'd probably behave similarly to the `MusicBrainzAlbumProvider`.

Or are you maybe asking about something else?

Reply:
I'm just curious if our current metadata fetcher can handle this, but it looks like it does not for now, but that is fine.